integration_tests/features/flow_run_middle_nodes.feature

```diff
@@ -41,3 +41,20 @@ Feature: `flow run` command with py nodes in the middle
       | middle_2.middle_script.py |
     And the script middle_2.middle_script.py output file has the lines:
       | Status: skipped |
+
+  Scenario: fal flow run command with set intersection selector
+    Given the project 008_pure_python_models
+    When the data is seeded
+
+    When the following command is invoked:
+      """
+      fal flow run --profiles-dir $profilesDir --project-dir $baseDir --experimental-models
+      """
+    And the following command is invoked:
+      """
+      fal flow run --profiles-dir $profilesDir --project-dir $baseDir --select model_a+,model_c+ --experimental-models
```

Comment:
Can we have a more explicit example of an intersection? Say we had dag like this:

```
          model_c
        /
model_a
        \
	  model_d
	/
model_b
        \
  	  model_e
```
in which case `fal flow run --select model_a+,model_b+` would only run model d and not c and e.

Reply:
I'll look for a similiar DAG from the existing projects, and add a test for that one too. Thanks!